Cython/Utility/ModuleSetupCode.c

```diff
@@ -2190,17 +2214,25 @@ static PyObject* __Pyx_PyCode_New(
     }
 
     #if CYTHON_COMPILING_IN_LIMITED_API
+    #if CYTHON_AVOID_UNSAFE_BORROWED_REFERENCES && PY_VERSION_HEX > 0x030d00b1
```

Comment:
```suggestion
    #if CYTHON_AVOID_UNSAFE_BORROWED_REFERENCES && __PYX_LIMITED_VERSION_HEX > 0x030d00b1
```

We should base it on the version of the Limited API it's compiled with rather than the version of Python it's compiled with.